kratos/sources/kratos_application.cpp

```diff
@@ -322,6 +322,10 @@ void KratosApplication::RegisterKratosCore() {
     KRATOS_REGISTER_FLAG(MARKER);
     KRATOS_REGISTER_FLAG(PERIODIC);
 
+    // Note: using internal macro for these two because they do not have a NOT_ version
+    KRATOS_ADD_FLAG_TO_KRATOS_COMPONENTS(ALL_DEFINED);
+    KRATOS_ADD_FLAG_TO_KRATOS_COMPONENTS(ALL_TRUE);
```

Comment:
wasn't this part of a different PR?

Reply:
That PR was a cherry-pick from this branch. It should go away once we merge master (will do later with some of the more "cosmetic" changes suggested).

Reply:
The alredy-merged changes should be gone now.